src/tests/react-testing-library.test.tsx

```diff
@@ -1,34 +1,114 @@
 import React from 'react';
+import { fireEvent, logRoles } from '@testing-library/dom';
 import { render, screen } from '@testing-library/react';
 import { App } from '../components/app';
+import { Toggle, ToggleProps } from '../components/toggle/toggle';
+import { Button, ButtonProps } from '../components/button/button';
 
 describe('React Testing Library do:', () => {
 
     test('create snapshot of mounted component', () => {
-        throw new Error('Not migrated yet');
+        // render app
+        render(<App />);
+
+        // check the snapshot
+        expect(screen).toMatchSnapshot;
     });
 
     test('find child by component', () => {
-        throw new Error('Not migrated yet');
+        // render App
+        render(<App />);
+        
+
+        // find the component
+        const childElement = screen.getByText(/click/i);
+
+        // check if it exists
+        expect(childElement).toBeInTheDocument();
     });
 
     test('find child by attribute', () => {
-        throw new Error('Not migrated yet');
+        // render App
+        const {container} = render(<App/>);
+
+        // find the component 
+        const childElement = container.querySelector('[data-test-target="button-element"]'); // had to use query selector because getByTestId doesn't work since that function requires a data attribute named data-testid.
```

Comment:
You can [set testId attribute name](https://dev.to/mbarzeev/custom-queries-for-react-testing-library-1974) as `data-test-target`, so `getByTestId` will use it to build a correct selector